packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec

```diff
@@ -6,12 +6,12 @@
 %global gem_name hammer_cli_katello
 %global confdir hammer
 
-%global release 3
-%global prerelease .pre.master
+%global release 1
+%global prerelease
```

Comment:
It's kinda silly the way it works but you can just comment this out or remove it from the spec file.